### PR TITLE
adding ability to add new resource

### PR DESCRIPTION
**Pull request by Michal-Shamir**
relating to issue #39 
in the existing course page - 
added the ability to add a new resource for the course

files added:
resource.html

files modified:
forms.py - defines a new form for submitting new resource
routes. course - added routing to new resource page
course.py - added a function to create a new resource for the course

-now the resource added isn't added to the database yet, needs to be handled. 
image

image

### CODE REVIEW

File: tutor/routes/home.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def index_route():
 def register_route():
     return register.register()
 
+
```
Comment (ifireball) [inline tutor/routes/home.py:15]: it doesn't seem like this empty line really needs to be added or belongs in this PR.

File: tutor/controllers/course.py [hunk 2]
```diff
@@ -22,3 +23,9 @@ def showCoursePage(id):
         {'id': 3, 'name': 'resource3', 'description': 'blala3', 'link': 'https://google.com'},
     ]
     return render_template('course.html', name=name, resources=resources)
+
```
Comment (ifireball) [inline tutor/controllers/course.py:26]: shouldn't resources-related controllers go in a `resources.py` file rather then a `courses.py` file?

File: tutor/routes/course.py [hunk 3]
```diff
@@ -5,3 +5,8 @@
 @app.route('/course/<id>')
 def get_course_info_route(id):
     return course.showCoursePage(id)
+
```
Comment (ifireball) [inline tutor/routes/course.py:8]: shouldn't resources-related routes go in a resources.py file rather then a courses.py file?

**Review by ifireball — COMMENTED** [review]
Code generally LGTM but I've some questions about chosen file names.

Also please document how you've tested this change or, preferably, add automated tests.

File: tutor/controllers/course.py (hunk 2 above)
Comment (yuvalmargules) [reply tutor/controllers/course.py:26]: we were discussing about it. we get into resources from the courses route once we choose a course, so we did it that way. but you're right, we may better do it separately.
@mosesnagar @Michal-Shamir

Comment (Michal-Shamir) [reply tutor/controllers/course.py]: @ifireball I splited the resources related routes and controllers .

**Review by ifireball — APPROVED** [review]
LGTM

**Pull request merged**